src/test/java/seedu/apollo/ParserTest.java

```diff
@@ -276,5 +275,12 @@ void parseShowMod_emptyDescription_expectsNoException() {
         assertDoesNotThrow(() -> Parser.getCommand(userCommand, ui, size, null));
     }
 
-
+    @Test
+    void parseHelpCommand_extraWord_expectException() throws UnexpectedException {
+        String userCommand = "help event hi";
+        Ui ui = new Ui();
+        int size = 1;
+        Command newCommand = Parser.getCommand(userCommand, ui, size, null);
+        assertNull(newCommand);
```

Comment:
Hmmm correct me if I'm wrong but if your test is expecting an exception, ideally you should use `assertThrows()` with the exception class you are expecting for the method.

Reply:
oh yes you are right. Thank you

Reply:
hmm but all the tests previously that throw UnexpectedException assertedNull so I followed it...Does that mean change for all that also?

Reply:
getCommand returns null so maybe thats why assertNull